stl/debugger/STL.natvis

```diff
@@ -22,6 +22,20 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
   </Type>
 
 
+  <Type Name="std::error_category">
+    <DisplayString Condition="_Addr == 1">[future]</DisplayString>
+    <DisplayString Condition="_Addr == 3">[generic]</DisplayString>
+    <DisplayString Condition="_Addr == 5">[iostream]</DisplayString>
+    <DisplayString Condition="_Addr == 7">[system]</DisplayString>
+    <DisplayString>@{((uintptr_t)this),x}</DisplayString>

```

Comment:
I think we should:
```suggestion
    <DisplayString>[{name(),sb}]</DisplayString>
    <Expand>
      <Synthetic Name="hint">
        <DisplayString>Enable "Allow Function Calls In Value Formatting" if you see "???" here</DisplayString>
      </Synthetic>
    </Expand>
```
here. It requires "Allow Function Calls In Value Formatting" to be enabled in the debugger, but produces an identical experience for library-defined and user-defined `error_category`s.